jigasi/rootfs/defaults/sip-communicator.properties

```diff
@@ -1,8 +1,10 @@
-{{ $ENABLE_TRANSCRIPTIONS := .Env.ENABLE_TRANSCRIPTIONS | default "0" | toBool -}}
 {{ $JIGASI_BREWERY_MUC := .Env.JIGASI_BREWERY_MUC | default "jigasibrewery" -}}
 {{ $JIGASI_XMPP_USER := .Env.JIGASI_XMPP_USER | default "jigasi" -}}
 {{ $JIGASI_JVB_TIMEOUT := .Env.JIGASI_JVB_TIMEOUT | default "30000" -}}
+{{ $JIGASI_MODE := .Env.JIGASI_MODE | default "sip" -}}
```

Comment:
Would it make sense to use separate files and generate the config based on the mode? It would de-clutter the config file...